examples/04_GSD_FeI2.jl

```diff
@@ -283,11 +283,11 @@ ax_top = Axis(fig[1,1],ylabel = "meV",xticklabelrotation=π/8,xticklabelsize=12;
 ax_bottom = Axis(fig[2,1],ylabel = "meV",xticks = (markers, string.(points)),xticklabelrotation=π/8,xticklabelsize=12)
 
 heatmap!(ax_top,1:size(is_interpolated,1), ωs, is_interpolated;
-    colorrange=(0.0,0.07),
+    colorrange=(0.0, 1.0),
 )
 
 heatmap!(ax_bottom,1:size(is_binned,1), ωs, is_binned;
-    colorrange=(0.0,0.05),
+    colorrange=(0.0, 1.5e-6),
```

Comment:
Did we accidentally introduce a Δω factor into binned intensities? These are already integrated quantities, so no Δω should appear, and the colorrange should not need to change.

Reply:
This was the figure that was already "broken" (i.e., appeared blank) before anything was added by this PR. In other words, this change was essentially unrelated to the rest of the PR.

@Lazersmoke I don't think any of the binning functions call `intensities_interpolated`, so they should not be affected by this PR, and all the binning tests still pass. But the binned results in the FeI2 tutorial appear appear like they already needed some plotting tweaks. See binned plot [here](https://sunnysuite.github.io/Sunny.jl/dev/examples/04_GSD_FeI2.html). The change here just tweaks the plotting limits so that the results show up.

Reply:
Per discussions on Slack, the binning intensity scale may have been broken in https://github.com/SunnySuite/Sunny.jl/pull/271, but is now fixed through the commits in https://github.com/SunnySuite/Sunny.jl/pull/273, which have been merged into this PR branch.